modules/swagger-annotations/src/main/java/io/swagger/oas/annotations/links/Link.java

```diff
@@ -38,28 +36,45 @@
 @Inherited
 public @interface Link {
   /**
-   * 
+   * The name of this link.
+   * @return the link's name
    **/
   String name() default "";
 
   /**
-   * 
+   * A relative or absolute reference to an OAS operation. This field is mutually exclusive of the operationId field, and must point to an Operation Object. Relative operationRef values may be used to locate an existing Operation Object in the OpenAPI definition.
+   * @return an operation reference
    **/
   String operationRef() default "";
 
   /**
-   * 
+   * The name of an existing, resolvable OAS operation, as defined with a unique operationId. This field is mutually exclusive of the operationRef field.
+   * @return an operation ID
    **/
   String operationId() default "";
 
   /**
-   * 
+   * Array of parameters to pass to an operation as specified with operationId or identified via operationRef.
+   * @return the list of parameters for this link
    **/
-  LinkParameters parameters() default @LinkParameters();
+  LinkParameter[] parameters() default {};
 
   /**
-   * 
+   * A description of the link. CommonMark syntax may be used for rich text representation.
+   * @return the link's description
    **/
   String description() default "";
+  
+  /**
+   * A literal value or {expression} to use as a request body when calling the target operation.
+   * @return the request body of this link
+   **/
+  String requestBody() default "";
```

Comment:
Although this has the same name, I believe it is different than the spec's definition of a `Request Body` object.  For a link, the `requestBody` field's description is  `A literal value or {expression} to use as a request body when calling the target operation.` which implies just a simple string, and not a full blow `Request Body` (which has a description, a required boolean and an array of content objects)